### DISCUSSION TITLE
Mac Terminal Recommendations

### DISCUSSION THREAD

**Discussion by joshualeond** [FedeClaudi/Term.jl#125] [Q&A] [answered]
I'm testing out `Term.jl` and the docs look amazing. However, when running in the default terminal on Mac it doesn't appear the same as what I'm seeing in the docs. Is there a recommended alternative terminal app when using `Term.jl` or am I perhaps just having issues on my side?

<img width="726" alt="image" src="https://user-images.githubusercontent.com/3802502/175542646-634d42b1-3785-4d0c-8f29-4c9593eb1858.png">

Also, is there a recommended way to view a docstring using `Term.jl`?

Thanks,
Josh

**Comment by FedeClaudi** [comment, marked as answer]
Hi,

different terminals handle ANSI codes and some characters differently. A fancier alternative like iTerm should be able to do better. If you use VS Code it’s built in terminal is also great.

`termshow(print)` will show all methods and doc string for print (or most of it anyways). Or if you have the doc string as a markdown object (MD) you can use parse_md to render it (need Term 1.0.0) for both

**Reply by joshualeond** [reply]
Thanks so much! I'm testing it out in VS Code and now I'm seeing what I expected and it looks amazing. 

One other question: Do you know why calling `termshow(Normal)` when using the `Distributions.jl` package returns the following instead of the docstring you get from the help mode in the REPL:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/3802502/175555526-e53dcd54-2d8a-4376-a787-53858dd5b2bd.png">

**Reply by FedeClaudi** [reply]
Nice! 

It shouldn’t! I’m on holiday for a few days but will check when I can.

**Reply by joshualeond** [reply]
Sounds good, thanks for your work... this is really cool.
